### DISCUSSION TITLE
How to remove the "GItlens:" prefix from the sidebar?

### DISCUSSION THREAD

**Discussion by ShubhamVerma1811** [gitkraken/vscode-gitlens#1721] [Q&A] [answered]
Hey,
I noticed that on my Windows VSCode  the sidebar of gitlens shows "Gitlens:"  as prefix, but not on Mac.
I would like to know if that prefix could be turned off by some settings?

Windows:
![windows](https://user-images.githubusercontent.com/25576658/141462638-0e248e88-2930-4a39-9ea9-a1d37ee810ad.png)

MacOS
<img width="345" alt="Screenshot 2021-11-12 at 10 46 38" src="https://user-images.githubusercontent.com/25576658/141462647-2413c822-403f-43e5-828b-61d45a432243.png">

Thanks for your time and efforts.

**Comment by eamodio** [comment, marked as answer]
VS Code adds the prefix to views when they aren't in their default location. You can reset a view to its default location by right-clicking on it and choosing "Reset Location". And if you want to quickly move GitLens views from the Source Control side bar to the GitLens side bar (or vice versa) you can use the `GitLens: Set Views Layout` command from the Command Palette.

**Reply by ShubhamVerma1811** [reply]
Finally!
Thank you!
